arcgis-ios-sdk-samples/Edit data/Edit with branch versioning/EditWithBranchVersioningViewController.swift

```diff
@@ -376,7 +376,7 @@ class EditWithBranchVersioningViewController: UIViewController {
                 object: textField,
                 queue: .main
             ) { _ in
-                let text = alertController.textFields![0].text!.trimmingCharacters(in: .whitespacesAndNewlines)
+                let text = textField.text!.trimmingCharacters(in: .whitespacesAndNewlines)
                 // Enable the done button if branch version textfield is not empty.
                 createAction.isEnabled = !text.isEmpty
```

Comment:
There's the same issue here with the observer holding a strong reference to `createAction` and `createAction` extending the lifetime of `textFieldObserver`. Please fix, like you did for `doneAction`, here and in the other samples (if necessary).

Reply:
🤦‍♂️  My bad, I didn't pay attention to the not changed lines...